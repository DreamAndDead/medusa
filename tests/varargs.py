def foo():
    pass

def varargs(name, *args):
    print("Name is: ", name)
    print(*args)

varargs("first", 1, 3, 6, 4, 7)
varargs("second", "hello", "python", "world")