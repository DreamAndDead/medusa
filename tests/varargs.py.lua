dofile('./pythonlua/luainit.lua')

local function foo()

end
local function varargs(name, ...)
    local args = list {...}
    print("Name is: ", name)
    print(unpack(args))
end
varargs("first", 1, 3, 6, 4, 7)
varargs("second", "hello", "python", "world")