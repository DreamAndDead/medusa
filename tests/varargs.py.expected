Name is: 	first
1	3	6	4	7
Name is: 	second
hello	python	world
